polyglot.js

```diff
@@ -2,27 +2,56 @@ class PolyGlot {
+				}
+		}
+
+		static get languages() {
+			switch(game.system.id.toLowerCase()){
+				default:
+					return CONFIG[game.system.id.toUpperCase()].languages;
+				case "wfrp4e":
+					let re = /Language \((.+)\)/i;
+					const pack = game.packs.get("wfrp4e.skills");
+					return pack.getIndex().then( itemList => {
+						let langs = {};
```

Comment:
`langs` should be set to `const` since the object is modified but the variable is not changed.